jina/peapods/pods/k8slib/kubernetes_tools.py

```diff
@@ -167,7 +165,9 @@ def get_port_forward_contextmanager(
 
     :return: context manager which sets up and terminates the port-forward
     """
-    _wait_for_gateway(namespace, timeout)
+    import portforward
+
+    _wait_for_flow_ready(namespace, timeout)
```

Comment:
this should not be implemented here. This should be part of every `K8sPod` `wait_for_ready` logic.

Reply:
Let's have that before this

Reply:
thanks removed